lewis/core/devices.py

```diff
@@ -112,16 +112,9 @@ def __init__(self, module):
 
         self._device_types = list(get_members(self._module, is_device).values())
 
-        if not self._device_types:
-            # syslog.warning('No devices discovered in module {}.'.format(module.__name__))
-            pass
-        else:
-            pass
-            # syslog.info('Discovered devices in module {}: {}'.format(
-            #    module.__name__, ','.join([x.__name__ for x in self._device_types])))
-
         submodules = get_submodules(self._module)
 
+        self._device_types += list(get_members(submodules.get('device'), is_device).values())
```

Comment:
I think our gut instincts agree, and I'd also say `devices`, because there might be more than one device type.